---
victor_hardware_interface/ControlModeStatus active_control_mode
bool has_active_control_mode
